Makefile

```diff
@@ -1,9 +1,9 @@
 APP_NAME=queequeg
-HOST=localhost
-PORT=3000
-SINGULARITY_HOST=localhost
-SINGULARITY_PORT=7099
-SINGULARITY_BASE=/singularity
+HOST?=localhost
```

Comment:
Conditionally set the variable. This way you can override them from the command line:

````bash
SINGULARITY_HOST=vagrant-singularity make server
````